### ISSUE TITLE
What are the loss functions that you have used for training the IR152, IRSE50, and MobileFace in the paper?

### ISSUE THREAD

**Issue by GeorgeCarpenter** [TencentYoutuResearch/Adv-Makeup#18]
Thanks.

**Comment by yubangji123** [comment]
Please clarify your question. Do you want to know the losses for training these face recognition models? Or you hope to know the adversarial losses when attacking those three models?

**Comment by GeorgeCarpenter** [comment]
The losses for training these face recognition models.

**Comment by yubangji123** [comment]
You can refer to this paper: https://openaccess.thecvf.com/content_CVPR_2019/papers/Deng_ArcFace_Additive_Angular_Margin_Loss_for_Deep_Face_Recognition_CVPR_2019_paper.pdf
